source/scss/components/_fti-select.scss

```diff
@@ -0,0 +1,43 @@
+.fti-select {
+  padding: 0;
+
+  >span {
+    color: $gray-400;
+    font-size: 12px;
+    font-weight: bold;
+    margin-left: 15px;
```

Comment:
Maybe you can change it in a future PR, in ETS-1382 we are introducing a variable that is using 15px for spacings, to keep them all together.

Reply:
Sure :)